### ISSUE TITLE
Safari: Top content doesn't scroll with page

### ISSUE THREAD

**Issue by epheterson** [msolters/Tiles#19]
Low-priority, though works well in Chrome. See video.

https://www.dropbox.com/s/qr00vndrxiy3414/safari%20scroll.mov?dl=0

**Comment by msolters** [comment]
Tempted to not fix this out of pure spite for Safari.  What the hell, man.  y cant safari just be cool?

**Comment by msolters** [comment]
So in this case I think it's more along the lines of -- vertical scrolling just should not be happening period!  I have noticed that most Apple browsers feel the need to let you drag and scroll the content past the edge.  The issue here is that the nav children are absolutely positioned to the corners or edges, everything else is relative.

**Comment by epheterson** [comment]
Yeah not the end of the world, though typically Safari does let you scroll the content to indicate you've reached the end.

**Comment by msolters** [comment]
@epheterson I think this is fixed???

**Comment by epheterson** [comment]
Not yet!
